pathogen-workflows.sql

```diff
@@ -14,6 +14,59 @@
  */
 set intervalstyle = 'iso_8601';
 
+/* Steampipe's conceit of "it's just tables and SQL" has a tendency to break
+ * down in the face of complex queries and reveal cracks in the abstraction.
+ * When the query planner really rolls up its sleeves and digs in, it can often
+ * shuffle around where filter conditions are applied in ways that _really
+ * matter_ to Steampipe's operation.  For example, it might move a filter
```

Comment:
How did you figure this out, just by looking at the execution plan??

Reply:
Yeah, looking at query execution plans and also what the Steampipe table in question requires/uses as list/get keys ([e.g.](https://github.com/nextstrain/steampipe-plugin-github/blob/80bfce2db51dd254b79fb138ed54c7a3c7e1e4eb/github/table_github_actions_repository_workflow_run.go#L18-L38)).